pkg/config/api-server/catalog/config.go

```diff
@@ -3,7 +3,8 @@ package catalog
 // Not yet exposed via YAML and env vars on purpose
 type CatalogConfig struct {
 	Bootstrap      BootstrapApiConfig
-	DataplaneToken DataplaneTokenApiConfig
+	DataplaneToken DataplaneTokenApiConfig // DEPRECATED: remove in next major version of Kuma
```

Comment:
> Jakub Dyszkiewicz
answering many questions about this dataplaneToken localurl:
previously the behavior was that if token server is disabled then verification is disabled and dataplaneToken.localUrl in catalog is empty and that’s how we could detect that verification is disabled in kuma-dp and kumactl.
now, we cannot turn off the admin server if verification is disabled and serve admin.localUrl empty because even when verification is turned off someone may use provided ca manager anyways.